lms/static/styles/components/_LMSFilePicker.scss

```diff
@@ -1,8 +1,6 @@
 .LMSFilePicker__dialog {
-  display: flex;
-  flex-direction: column;
-  max-height: 100vh;
+  /* Prevent the Dialog from jumping size when changing state from fetching to fetched. */
+  min-height: 400px;
```

Comment:
I found a regression here. In production, the dialog will shrink if the iframe is vertically smaller than ~500px:

<img width="580" alt="Screenshot 2019-08-27 17 08 35" src="https://user-images.githubusercontent.com/2458/63788442-66dbdc80-c8ed-11e9-974b-0c8de2ccc038.png">

Locally that doesn't work:

<img width="608" alt="Screenshot 2019-08-27 17 01 46" src="https://user-images.githubusercontent.com/2458/63788484-79eeac80-c8ed-11e9-9342-f28180b74c5b.png">

This is what `max-height: 100vh` achieved previously. That caused problems when the content height exceeded `100vh` though.